[Workers]: Update Form Submission Tutorial (#9335)
cloudflare/cloudflare-docs: content/workers/tutorials/handle-form-submissions-with-airtable/index.md

If you would like to follow along with this example, you can directly copy the `<form>` code from the [`Form` component](https://github.com/cloudflare/workers-airtable-form/blob/main/frontend/src/Form.js) into your own project, or use the codebase and plug in your own serverless function, following the next section in the tutorial.

## Step 2: Create a serverless function

> To handle the form submission, you will create and deploy a Workers serverless function that parses the incoming form data and prepares it for submission to Airtable.

Reviewer: ```suggestion
To handle the form submission, you will create and deploy a Worker that parses the incoming form data and prepares it for submission to Airtable.
```

{{</Aside>}}

> ## Step 3: Configure an Airtable base

Reviewer: ```suggestion
## 3. Configure an Airtable base
```

2.  **Airtable Base ID:** the alphanumeric base ID found at the top of your base's API page.
3.  **Airtable API Key:** the private API key found in example API requests on the Airtable API documentation page.

> ## Step 4: Submit data to Airtable

Reviewer: ```suggestion
## 4. Submit data to Airtable
```

With your Airtable base set up, and the keys and IDs you need to communicate with the API ready, you will now set up your Workers function and persist data from your form into Airtable.

> In `index.js`, begin by setting up a Workers handler that can respond to requests. When the URL requested has a pathname of `/submit`, you will handle a new form submission, otherwise, you will redirect to `FORM_URL`, a constant representing your front-end form URL (for example, [airtable-form-example.pages.dev](https://airtable-form-example.pages.dev)):

Reviewer: Per C3, 'index.js' would've been replaced by 'worker.js'.
